### DISCUSSION TITLE
I opened a ticket with github support (ticket number #1947625)

### DISCUSSION THREAD

**Discussion by loganaden** [community/community#43217] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

I haven't received any reply about this from the github team and it's been more than 24 hours.

**Comment by LuisContrerasGlz** [comment, marked as answer]
The length of time it takes for GitHub Support to respond to a ticket can vary depending on the volume of requests they are receiving and the complexity of the issue. it's a good idea to allow at least a few business days for a response
If you haven't received a response after a few business days, you can try reaching out to GitHub Support again to see if there has been any progress on your ticket. You can do this by replying to the original ticket or by contacting GitHub Support through the "Contact support" form on the GitHub Support site. Include your ticket number and a brief summary of your issue in your message.
